cypress/e2e/testtab.cy.js

```diff
@@ -1,16 +1,23 @@
 describe('About the Test tab', () => {
-  beforeEach(() => {
-  });
-
   beforeEach(() => {
     cy.clearDebugStore();
     cy.createReport();
     cy.createOtherReport();
     cy.initializeApp();
+    let storageIds = [];
```

Comment:
Please make this const, the variable is not reassigned so it should not be let.

Reply:
Gedaan.